tests/test_app.py

```diff
@@ -3,13 +3,34 @@
 import os
 from unittest import mock
 
-import openai
 import pytest
 import quart.testing.app
+from httpx import Request, Response
+from openai import BadRequestError
 
 import app
 
 
+def fake_response(http_code):
+    return Response(http_code, request=Request(method="get", url="https://foo.bar/"))
```

Comment:
I'm confused by foo.bar, how is that connected to the tests?

Reply:
It isn't, but it is a required property of the Request option so I had to put something there